Saule/Http/HttpConfigExtensions.cs

```diff
@@ -48,15 +48,44 @@ public static void ConfigureJsonApi(this HttpConfiguration config, JsonApiConfig
             JsonApiConfiguration jsonApiConfiguration,
             bool overwriteOtherFormatters)
         {
+            if (overwriteOtherFormatters)
+            {
+                ConfigureJsonApi(config, jsonApiConfiguration, ConfigureFormattersEnum.OverwriteOtherFormatters);
+            }
+            else
+            {
+                ConfigureJsonApi(config, jsonApiConfiguration, ConfigureFormattersEnum.AddFormatterToStart);
+            }
+        }
+
+        /// <summary>
+        ///  Sets up serialization and deserialization of Json Api resources.
+        /// </summary>
+        /// <param name="config">The <see cref="HttpConfiguration"/> that is used in the setup of the application.</param>
+        /// <param name="jsonApiConfiguration">Configuration parameters for Json Api serialization.</param>
+        /// <param name="configureFormatters">
+        /// AddFormatterToStart - Formatter will be inserted at the start of the collection.
```

Comment:
This should be pretty clear from the enum values; maybe something like `Determines the relative position of the JSON API formatter`?